src/tls_openssl.c

```diff
@@ -20,11 +20,10 @@ static int mg_tls_err(struct mg_tls *tls, int res) {
   return err;
 }
 
-static STACK_OF(X509_INFO) * load_ca_certs(const char *ca, int ca_len) {
-  BIO *ca_bio = BIO_new_mem_buf(ca, ca_len);
-  if (!ca_bio) return NULL;
```

Comment:
Is this safe ? No longer checking for NULL before calling (nor freeing later)

Reply:
Fixed